RecoMuon/L2MuonSeedGenerator/src/L2MuonSeedGeneratorFromL1T.cc

```diff
@@ -41,6 +43,28 @@ using namespace std;
 using namespace edm;
 using namespace l1t;
 
+//--- Functions used in output sorting
+bool SortByL1Pt(L2MuonTrajectorySeed &A, L2MuonTrajectorySeed &B) {
+  l1t::MuonRef Ref_L1A = A.l1tParticle();
+  l1t::MuonRef Ref_L1B = B.l1tParticle();
+  return (Ref_L1A->pt() > Ref_L1B->pt());
+};
+
+bool SortByL1QandPt(L2MuonTrajectorySeed &A, L2MuonTrajectorySeed &B) {
```

Comment:
Method names should start with lower case letters, upper case being reserved to class names